<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>532</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Kutegram</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SlidingStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="authPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/client160.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Kutegram</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Crossplatform Telegram client, based on Qt.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SlidingStackedWidget" name="introStackedWidget">
          <property name="lineWidth">
           <number>0</number>
          </property>
          <widget class="QWidget" name="introPage">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>253</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="introButton">
              <property name="text">
               <string>Log in</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="phonePage">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Please, choose your country code and enter your phone number.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="countryBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="phoneEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="codeButton">
              <property name="text">
               <string>Send code</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="codePage">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>364</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Enter a confirmation code:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="codeEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="loginButton">
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="changeLabel">
              <property name="text">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;kutegram://change-phone&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Change phone number...&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chatPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabBar" name="tabBar" native="true">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="dialogView">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QMenu" name="actionsMenu">
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="logoutAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQt"/>
   </widget>
   <addaction name="actionsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="loginAction">
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="logoutAction">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About Kutegram</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>qtabbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialogView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>dialogView_activated(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logoutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>logoutAction_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quitAction_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutAction_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutQt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>introButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>introButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loginButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>codeButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeLabel</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeLabel_linkActivated(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dialogView_activated(QModelIndex)</slot>
  <slot>logoutAction_triggered()</slot>
  <slot>quitAction_triggered()</slot>
  <slot>aboutAction_triggered()</slot>
  <slot>aboutQt_triggered()</slot>
  <slot>introButton_clicked()</slot>
  <slot>codeButton_clicked()</slot>
  <slot>loginButton_clicked()</slot>
  <slot>changeLabel_linkActivated(QString)</slot>
 </slots>
</ui>
